demisto_sdk/commands/common/tests/script_test.py

```diff
@@ -338,6 +338,17 @@ def test_is_valid_subtype(self, current_file, answer):
         (V2_WRONG_DISPLAY, False),
     ]
 
+    IS_SKIPPING_DOCKER_CHECK = [("Packs/ApiModules", False, True),
+                                ("Packs/ApiModules", False, True),
```

Comment:
The first two cases seem identical, am I missing anything?

Reply:
You right, my mistake. changed it